src/utils/runtime/src/lib.rs

```diff
@@ -180,6 +180,12 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
     if query_log_path.is_ok() || ENABLE_QUERY_LOG_FILE {
         let query_log_path = query_log_path.unwrap_or(".risingwave/log".to_string());
```

Comment:
`.risingwave/log` seems to be designed for risedev. How about just writing the log files to `./` and set `RW_QUERY_LOG_PATH` in risedev?

Reply:
Hmm, one additional thing: 

https://github.com/risingwavelabs/risingwave/blob/a707bc215f69468e793f2d6fa34a1eb62ba9fec4/src/utils/runtime/src/lib.rs#L132-L133

This is a little bit dangerous, especially considering we are going to enable slow query logs in Cloud. I think we had better simply use 2 independant vairables:

1. If `RW_QUERY_LOG_PATH` is set, use it as path for log files (both query log and slow query log).
2. If `ENABLE_QUERY_LOG_FILE` is set to `true`, enable query log. (Slow query is always enabled)

Reply:
cc. @xxchan

Reply:
LGTM. BTW, the query log feat was added just for ad-hoc debugging without deliberations. 😇

So any enhancements to make it more mature should be good.

Reply:
One little enhancement is: why `RW_QUERY_LOG_PATH` has a prefix `RW_` but `ENABLE_QUERY_LOG_FILE` does not 🤣

Reply:
`ENABLE_QUERY_LOG_FILE` 
it is a constant

If `env` variable, then `RW_` I suppose